tomcat/src/main/java/com/linecorp/armeria/server/tomcat/TomcatService.java

```diff
@@ -175,43 +162,39 @@ public static TomcatService forFileSystem(Path docBase) {
      * Creates a new {@link TomcatService} from an existing {@link Tomcat} instance.
      * If the specified {@link Tomcat} instance is not configured properly, the returned {@link TomcatService}
      * may respond with '503 Service Not Available' error.
+     *
+     * @return tomcat service, will not manage the tomcat.
      */
     public static TomcatService forTomcat(Tomcat tomcat) {
```

Comment:
`a new {@link TomcatService}, which will not manage the specified {@link Tomcat}`